.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+##### SUMMARY
+<!--- Describe the change below, including rationale and design decisions -->
+
+##### GitHub Issues
+List the GitHub issues impacted by this PR:
+
+| GitHub Issue # |
+| -------------- |
+| |
+
+##### ISSUE TYPE
+<!--- Pick one below and delete the rest -->
+- Bugfix Pull Request
+- Docs Pull Request
+- Feature Pull Request
+- Test Pull Request
+
+##### COMPONENT NAME
+<!--- Write the short name of the module, plugin, task or feature below -->
+
+##### OUTPUT
+<!--- Paste the functionality test result below -->
+```paste below
+
+```
+##### ADDITIONAL INFORMATION
+<!--- Include additional information to help people understand the change here -->
+<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->
+
+<!--- Paste verbatim command output below, e.g. before and after your change -->
+```paste below
+
+```
+##### Checklist:
+
+- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
```

Comment:
What does "vetting" mean in this requirement? Is there a recommended procedure to check for "security issues" (or is this a judgement call for the PR submitter)?